src/locales/ko/mystery-encounters/dancing-lessons-dialogue.json

```
@@ -1,27 +1,27 @@
 {
-  "intro": "An {{oricorioName}} dances sadly alone, without a partner.",
-  "title": "Dancing Lessons",
-  "description": "The {{oricorioName}} doesn't seem aggressive, if anything it seems sad.\n\nMaybe it just wants someone to dance with...",
-  "query": "What will you do?",
+  "intro": "{{oricorioName}}[[가]] 외롭게 혼자 춤을 추고 있다.\n주변에 파트너도 없는 것 같다!",
```

```suggestion
  "intro": "{{oricorioName}}[[가]] 외로이 혼자 춤을 추고 있다.\n주변에 파트너도 없는 것 같다!",
```
의미는 달라지는 게 없는데 뭔가 조금 더 나은 듯하여 제안해 봅니다..